### ISSUE TITLE
A11y_.NET Core MAUI_Recipe App_Edit Recipe_Screenreader : The "Review" Label is not announced by Narrator and NVDA Screenreader

### ISSUE THREAD

**Issue by Ashutoshsahoo164** [dotnet/maui#15663]
### GitHubTags:
#A11yMAS;#A11yTCS;#BM_.NET Core[MAUI]-Win32-May2023;#.NET Core;#A11ySev2;#WCAG1.3.1;#DesktopApp;#Win11;#Benchmark;#Narrator;#NVDA;

### Actual:

While Navigating using Narrator screen reader , the "Review" label is not announced by it.

Note : This bug is also reproducing with NVDA Screen reader.

### Expected:
The Screen readers should announce the "Review" label when we navigate the control using screen readers. 

### User Impact:
Screen reader users will not be getting proper information about the Review control.

### Steps to Reproduce

1. Install and open ".NET MAUI" app.
2. Tab till "search box" and enter recipe name and search.
3. "Search result" page will open.
4. Tab till any recipe and select.
5. "Recipe" Screen will open.
6. Tab till Edit Recipe and Hit enter
7. Turn on Narrator and Verify that the bug by navigating from top of the page.

### Attachment : 

Narrator : 
[Recording-20230614_195645.webm](https://github.com/dotnet/maui/assets/85863895/e0dc7821-3ddb-42cd-8569-9ddbcdfc8184)

### Link to public reproduction project repository

Test

### Version with bug

7.0.49

### Last version that worked well

6.0

### Affected platforms

Windows

### Affected platform versions

Windows11 23H2

**Comment by rachelkang** [comment]
Hi, @Ashutoshsahoo164 - to my understanding, this is consistent with behaviors in all Windows apps. Only interactive controls can be reached when navigating through the controls via the Tab key. Could you share how you are navigating to the label? And what the expectation is, given how the platform behaves?

**Comment by Ashutoshsahoo164** [comment]
Hi @rachelkang  The expected here is , the screen readers should announce the Label name automatically when the focus lands to the slider. The screen reader should announce "Review Slider 80%"

**Comment by PoojaNamde** [comment]
#A11ypreview;

Added A11ypreview tag

**Comment by PoojaNamde** [comment]
@rachelkang Could you please close this issue as I am not getting close option.

#Regressed:09-22-23; #Closed;

Verified this issue in latest environment, issue is fixed hence closing the bug.
![image](https://github.com/dotnet/maui/assets/90901519/74e9697f-5e02-4e44-bcb9-235a9daf98dc)
